lib/rpush/client/redis/webpush/app.rb

```diff
@@ -0,0 +1,15 @@
+module Rpush
+  module Client
+    module Redis
+      module Webpush
+        class App < Rpush::Client::Redis::App
+          include Rpush::Client::ActiveModel::Webpush::App
+
+          def vapid_keypair=(value)
```

Comment:
How come this isn't required for active_record?

Reply:
I don't know, but without this, the Redis tests fail. The Pushy code has the same workaround. It looks like for AR, the `alias_attribute` call from the common module creates both reader and writer, while with Redis, it fails to create the writer method.